subsys/input/input_utils.c

```diff
@@ -119,6 +265,12 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 		      "usage: dump <on|off>",
 		      input_cmd_dump, 2, 0),
 #endif /* CONFIG_INPUT_EVENT_DUMP */
+#ifdef CONFIG_INPUT_KBD_MATRIX_STATE
```

Comment:
Can you add some documentation that guides the user on how to use and interpret the results from this command?